commit 93e997d442d617226c3dc307719e9b752f037e91
Author: Florian Schmaus <flo@geekplace.eu>
Date:   Thu Dec 19 10:58:31 2013 +0100

    lib/build.xml: Also set javac source to java6
    
    "javac: target release 6 conflicts with default source release 1.7"
    error will be emitted if a java7 compiler is used and target is set to 6
    but not source.

diff --git a/lib/build.xml b/lib/build.xml
index 066dfff..d480d57 100644
--- a/lib/build.xml
+++ b/lib/build.xml
@@ -20,6 +20,7 @@
 	srcdir="src/"
 	destdir="build/"
 	bootclasspath="${sdk-location}/platforms/${android.version}/android.jar"
+	source="6"
 	target="6"
 	/>
     <jar
